### ISSUE TITLE
room assistant companion does not start transmitting after restart

### ISSUE THREAD

**Issue by gadric** [mKeRix/room-assistant-companion-ios#5]
I have successfully changed from Bluetooth Classic to BLE and startet using the Companion App on my iPhone. 

The App works good and the room detection is good. But I have experience now a couple of times that after restart of my iPhone room assistant shows `not_home`

I have to open the companion app again in order to make it work.

**Comment by mKeRix** [comment]
I'm seeing the same issue, haven't thought of a good way around it yet though. The app already implements the APIs that Apple documents for restoring Bluetooth state after crashes/reboots, but it doesn't seem to work properly for reboots (anymore?). Unfortunately, Apple also does not allow the advertising state to be changed in the background as far as I read. As a quick workaround it may be nice to send a notifications when this occurs, so that it's quicker to notice & act. For a more long term solution I had a new idea for tracking iOS devices that wouldn't suffer from this problem, but that one is still very unproven and would also require a lot of changes.

Anyway, thanks for reporting!

**Comment by gadric** [comment]
I see. Let know if I can be of any assistance (e.g., testing etc)

**Comment by blacknell** [comment]
I don't believe is happening only on an iPhone app restart. Leaving it on in the background, after a while, it also stops advertising - rendering it invisible.

**Comment by Matsuo3rd** [comment]
I just added an automation from the iOS Shortcut app in order to launch Room Assistant iOS app upon geofence Arrival trigger (location  = home). This shall secure Room Assistant app is running when I get back home. Not ideal / little bit naggy but shall make the job. Additional trigger are also available (such as when iPhone is charging mode is on/off).
